src/ol3cesium.js

```diff
@@ -11,12 +11,26 @@ goog.require('olcs.VectorSynchronizer');
 
 
 /**
- * @param {!ol.Map} map
+ * @param {!(ol.Map|olcsx.OLCesiumOptions)} arg1 `ol.Map` or options
  * @param {Element|string=} opt_target Target element for the Cesium container.
+ *   Overrides a target set in options.
  * @constructor
  * @api
  */
-olcs.OLCesium = function(map, opt_target) {
+olcs.OLCesium = function(arg1, opt_target) {
```

Comment:
Allowing both ways does not add significant overhead, so I'm inclined to not deprecate the current signature. It also adds convenience if you are not interested in custom synchronizers.